xml/System.Linq/ParallelEnumerable.xml

```diff
@@ -3601,6 +3601,7 @@
         <exception cref="T:System.ArgumentNullException">
           <paramref name="source" /> or <paramref name="action" /> is a null reference (Nothing in Visual Basic).</exception>
         <exception cref="T:System.AggregateException">One or more exceptions occurred during the evaluation of the query.</exception>
+        <exception cref="T:System.NotSupportedException">The exception that occurs when this method is called.</exception>
```

Comment:
Since this overload always throws Not SupportedException on both .NET Framework and .NET Core, the other exceptions (on line 3600-3603) should be removed.

Reply:
Nice catch. Makes sense. Will remove both instances. Thanks.